NonEmergencyAmbulanceTransportation/R4/resources/Questionnaire-R4-NonEmergencyAmbulanceTransportationProgressNote.json

```diff
@@ -0,0 +1,551 @@
+              "required": false,
+              "linkId": "8.4.3",
+              "text": "Moderate to severe pain on movement"
+            }
+          ]
+        },
+        {
+          "type": "group",
+          "code": [],
+          "required": false,
+          "linkId": "8.5",
+          "text": "Wound",
+          "item": [
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.5.1",
+              "text": "Unable to sit in chair or wheelchair due to Grade II or greater decubitus ulcers on buttocks"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.5.2",
+              "text": "Chronic wounds requiring immobilization"
+            }
+          ]
+        },
+        {
+          "type": "group",
+          "code": [],
+          "required": false,
+          "linkId": "8.6",
+          "text": "Attendant required during transport",
+          "item": [
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.1",
+              "text": "Morbid obesity requires additional personnel/equipment to handle"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.2",
+              "text": "Third party attendant required to regulate or adjust oxygen en route"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.3",
+              "text": "Special handling en route – isolation"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.4",
+              "text": "IV medications/fluids required during transport"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.6.5",
+              "text": "Restraints (physical or chemical) anticipated or used during transport"
+            }
+          ]
+        },
+        {
+          "type": "group",
+          "code": [],
+          "required": false,
+          "linkId": "8.7",
+          "text": "Mental",
+          "item": [
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.7.1",
+              "text": "Danger to self or others"
+            },
+            {
+              "type": "boolean",
+              "code": [],
+              "required": false,
+              "linkId": "8.7.2",
+              "text": "Confused, combative, lethargic, comatose"
+            }
+          ]
+        },
+        {
+          "type": "boolean",
```

Comment:
Should provide a text box for the provider to add note for Other.

Reply:
Will add that in. Thanks!